data/example-data-mongodb-book/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
move the comment to the very top of the file

Reply:
comment has to be within the <?xml> tag in order to work